packages/daemon/src/daemon.js

```diff
@@ -252,6 +252,14 @@ const makeDaemonCore = async (
   /** @type {import('./types.js').WeakMultimap<Record<string | symbol, unknown>, string>['get']} */
   const getIdForRef = ref => idForRef.get(ref);
 
+  /** @type {import('./types.js').Provide} */
+  const provide = id =>
```

Comment:
Just for explicitness:
```suggestion
  const provide = (id, _formulaType) =>
```